src/view/edit.js

```diff
@@ -1,45 +1,61 @@
 (function(){
-  LifeGamification.edit = {};
-
-  const skillEditHTML = function (number) {
-    return (`
-    	<div class="skill">
-        <a class="skill__remove"><img src="../../assets/x.svg"class="skill__remove" id="remove${number}"></a>
-      `) + LifeGamification.home.skillHTML(number);
-  }
-
-  const appendEditSkill = function (skill) {
-    $('.all-skills').append(skillEditHTML(LifeGamification.skillsView.length));
-    LifeGamification.skillsView.push(skill);
-    LifeGamification.home.viewLevelAndExp(skill);
-  }
-
-  LifeGamification.edit.render = function (skills) {
-    for (let name in skills) {
-      appendEditSkill(skills[name]);
+  class EditSkill extends  LifeGamification.view.SkillView {
+    _addRemoveButton() {
+      const removeButton = this._appendNewElement(
+        'img', {
```

Comment:
`removeButton` should be a `button` containing this image as background-image in CSS or it could nest `img` inside.